Fatras/include/ActsFatras/EventData/ProcessType.hpp

```diff
@@ -18,6 +18,7 @@ namespace ActsFatras {
 /// Encodes the type of process that generated a particle.
 enum class ProcessType : uint32_t {
   eUndefined = 0,
+  ePhotonConversion = 1,
 };
```

Comment:
I think this might be too much, at least right now. Since this is a fast sim, we don't expect to have too many different effects. Otherwise, this can be fixed if this enum becomes too large. But then, a better grouping strategy might become obvious. So I would leave it as it is for now.

Reply:
Ok.